cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -203,25 +288,23 @@ int main(int argc, char** argv)
             }
             i++;
         }
-        // NOTE: If number of windows smaller than batch capacity, then run POA generation
-        // once last window is added to batch.
-        if (status == StatusType::exceeded_maximum_poas || (i == get_size(windows) - 1))
-        {
-            // No more POA groups can be added to batch. Now process batch.
-            process_batch(batch.get(), msa, print);
-
-            // After MSA is generated for batch, reset batch to make roomf or next set of POA groups.
-            batch->reset();
-
-            std::cout << "Processed windows " << window_count << " - " << i << std::endl;
-            window_count = i;
-        }
 
         if (status != StatusType::exceeded_maximum_poas && status != StatusType::success)
         {
             std::cerr << "Could not add POA group to batch. Error code " << status << std::endl;
+            error_count++;
+            if (error_count > get_size(windows))
+                break;
         }
     }
 
+    if (print && long_read)
```

Comment:
we should add a `print_graph` option, and print it for both long and short read

Reply:
do we really need to add print graph option for short reads? the current sample generates 1000 poa groups, that would be 1000 poa graphs.

Reply:
although keeping symmetry between short-read and long-read samples is preferred, but considering large number of graphs for short-read sample, I modified the help message to better reflect behavior of -p for short-read and long-read.

Reply:
alternatively we can add this print-graph option and allow its use only along -l flag. I think this is better, will change it that way.